@here/harp-mapview/lib/poi/PoiRenderer.ts

```diff
@@ -539,7 +538,10 @@ export class PoiRenderer {
 
         if (opacity > 0) {
             if (!poiInfo.buffer) {
-                this.preparePoi(poiInfo.textElement, env);
+                const { imageItem, imageTexture } = this.imageCached(poiInfo.imageTextureName);
```

Comment:
ditto, why do we need to move this here? Furthermore, now it's in two places, prepareRender() and addPoi()